src/SideKicks.jl

```diff
@@ -1,3 +1,12 @@
+"""
+Main module for `SideKicks.jl` -- a analysis package for performing parameter inference on compact object stellar binaries.
+
+X functions are exported from this module for public use:
+
+(RTW - how to find these?)
```

Comment:
What do you mean by this?

Reply:
FYI, julia does not exactly make a distinction between private/public as other codes. The modules are namespaces, exporting something simply makes it available without using the namespace before it, but you can still access everything if you specify the namespace. The next julia versions will have a `public` keyword which will help identify what things are meant to be public API and which are internal and not meant for users. I dunno exactly how that one will work though.

https://docs.julialang.org/en/v1.12-dev/base/base/#public

Reply:
I don't remember exactly what I was looking for here, but I think I was inspired by the layout in the [Documenter documentation page](https://documenter.juliadocs.org/stable/lib/public/#Documenter). 

They have a list there of exports, and I think I was trying to replicate in here, but couldn't figure out how. Very low priority, but possibly useful later.